File: apps/beny-bot/src/commands/benyBoy.ts
import type { ChatInputCommandInteraction } from "discord.js";
import { CommandsEnum } from "./types";

async function action(interaction: ChatInputCommandInteraction) {
  await interaction.reply("Boy!");
}

export const benyBoyCommand = {
  action,
  command: CommandsEnum.BENY,
  description: "Beny Boy!",
};

File: apps/api/internal/bot/commands/benyboy.go
package commands

import "github.com/bwmarrin/discordgo"

var BenyBoyCommand = Command{
	Name:        "beny",
	Description: "Beny!",
	Action: func(s *discordgo.Session, i *discordgo.InteractionCreate, deps *BotDeps) error {
		return replyPublic(s, i, "Boy!")
	},
}
